src/NuGet.Core/NuGet.PackageManagement/Audit/AuditChecker.cs

```diff
@@ -168,22 +216,60 @@ internal static void CreateWarningsForPackagesWithVulnerabilities(Dictionary<Pac
                         package.Version.ToNormalizedString(),
                         severityLabel,
                         vulnerability.Url);
-                    var restoreLogMessage =
-                        RestoreLogMessage.CreateWarning(logCode,
-                        message,
-                        package.Id);
-                    logger.Log(restoreLogMessage);
+
+                    bool counted = false;
+                    for (int i = 0; i < auditInfo.Projects.Count; i++)
+                    {
+                        string projectPath = auditInfo.Projects[i];
+                        auditSettings.TryGetValue(projectPath, out (bool, PackageVulnerabilitySeverity) auditSetting);
+
+                        if (auditSetting == default || auditSetting.Item1 && (int)vulnerability.Severity >= (int)auditSetting.Item2)
```

Comment:
Please use named Tuples, I'm not a fan of seeing `Item1`, `Item2`, `Item3` etc

Reply:
TryGetValue was finnicky last time I tried, but I was able to figure out the syntax file. It looks a bit dumb, but it works :D